__test__/pet.test.js

```diff
@@ -0,0 +1,132 @@
+const Pet = require ('../src/pet.js')
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet ('Bob')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
+        const pet = new Pet('Bob');
+        expect(pet.name).toEqual('Bob');
+      });
+    it('has an initial age of 0', () => {
+      const pet = new Pet('Bob')
+      pet.growUp()
+      expect(pet.age).toEqual(1);
+    })
+    it('has inital hunger of 0', () => {
+      const pet = new Pet ('Bob')
+      pet.getHungry()
+      expect(pet.hunger).toEqual(5)
+    })
+    it('has inital fitness of 10', () => {
+      const pet = new Pet ('Bob')
+      pet.becomeUnfit()
+      expect(pet.fitness).toEqual(7)
+    })
+    });
```

Comment:
The formatting is off here. As above the closing bracket of any of your functions should be at the same level as the opening one, there should be a clear line between each of your tests, there should be clear line before the assertions begin, and the amount of indentation used should be uniform throughout. As it is, sometimes you have 2 spaces and sometimes you have 4. Ive done this one as an example for you, would be great if you could apply this to the rest of the file.

```
describe('constructor', () => {
    it('returns an object', () => {
        expect(new Pet ('Bob')).toBeInstanceOf(Object);
    });

    it('sets the name property', () => {
        const pet = new Pet('Bob');
        expect(pet.name).toEqual('Bob');
    });

    it('has an initial age of 0', () => {
        const pet = new Pet('Bob')
        pet.growUp()

        expect(pet.age).toEqual(1);
    });

    it('has inital hunger of 0', () => {
        const pet = new Pet ('Bob')
        pet.getHungry()

        expect(pet.hunger).toEqual(5)
    });

    it('has inital fitness of 10', () => {
        const pet = new Pet ('Bob')
        pet.becomeUnfit()

        expect(pet.fitness).toEqual(7)
    });
});
```